spec/acceptance/sqlserver_config_spec.rb

```diff
@@ -2,34 +2,29 @@
 require 'securerandom'
 require 'erb'
 
-host = find_only_one('sql_host')
-
 # Get instance name
 inst_name = ('MSSQL' + SecureRandom.hex(4)).upcase
-
 # Get database name
 db_name   = ('DB' + SecureRandom.hex(4)).upcase
 
-describe 'sqlserver::config test', node: host do
+describe 'sqlserver::config test' do
   def ensure_sqlserver_instance(inst_name, ensure_val = 'present')
-    create_new_instance = <<-MANIFEST
+    pp = <<-MANIFEST
       sqlserver_instance{'#{inst_name}':
       ensure                => '#{ensure_val}',
       source                => 'H:',
-      features              => [ 'SQL' ],
+      features              => ['DQ', 'FullText', 'Replication', 'SQLEngine'],
```

Comment:
Was this to fix a cross-version bug/difference?

Reply:
Yeh, the  'SQL' grouping is no longer present on the newer versions with the specific features having to be explicitly stated instead.